x-pack/plugins/synthetics/public/legacy_uptime/components/settings/add_connector_flyout.tsx

```diff
@@ -61,12 +43,10 @@ export const AddConnectorFlyout = ({ focusInput, isDisabled }: Props) => {
           setAddFlyoutVisibility(false);
           focusInput();
         },
-        supportedActionTypes: (actionTypes ?? []).filter((actionType) =>
-          ALLOWED_ACTION_TYPES.includes(actionType.id as ActionTypeId)
-        ),
+        featureId: 'uptime',
```

Comment:
The synthetics plugin doesn't have an existing dependency on the actions plugin and I didn't think it made sense to add one just so I could use the `UptimeConnectorFeatureId` constant.